CONTRIBUTING.md

```diff
@@ -99,16 +99,17 @@ For Windows you can use below command:
 $env:IASQL_ENV = 'local'; docker-compose up --build
 ```
```

Comment:
the entire architecture section that mentions dblink and the graphite worker also needs to be updated